source/ceylon/time/Date.ceylon

```diff
@@ -32,6 +32,15 @@ shared interface Date
 
     "Returns the [[DateRange]] between this and given [[Date]]."
     shared formal DateRange rangeTo( Date other );
+    
+    "Two `Date`s are considered equals if they have the same 
```

Comment:
I didnt understand, in my mind just say that two Date's must be equals _is_ the same as implicit equals. If I understand correctly we should say _how_ (in the sense of what define two Date to be considered the same from others view). So, anyone with implementations of Date would know what expect from something like `Boolean isSame(Date a, Date b)` .. or i got it wrong?